packages/sdk-react/src/BaseDashboard.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from "react";
+
+import { mergeStyles } from "@fluentui/react";
```

Comment:
Could we use [mergeClasses](https://master--628d031b55e942004ac95df1.chromatic.com/iframe.html?viewMode=docs&id=concepts-developer-styling-components--page#merging-component-styles) from `@flunetui/react-components` instead of `@fluentui/react`? This will help to reduce the dependency of this package.

Reply:
This is because I need to generate a class name in a class component, but `makeStyles` or `mergeClasses` from `@flunetui/react-components` is a react hook that can only be used in the function component, so there is no way but to use the `mergeStyles`.